### PR TITLE
don't restrict det_pylibdir to only EB-provided Python, there's no reason to

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review

### CODE REVIEW

File: easybuild/easyblocks/generic/pythonpackage.py [hunk 1]
```diff
@@ -53,27 +53,24 @@
...
-            raise EasyBuildError("Last line of output of %s does not start with specified prefix %s: %s (exit code %s)",
-                                 cmd, prefix, out, ec)
-
-        pylibdir = txt[len(prefix):]
-        log.debug("Determined pylibdir using '%s': %s", cmd, pylibdir)
-        return pylibdir
+
+    # determine Python lib dir via distutils
+    # use run_cmd, we can to talk to the active Python, not the system Python running EasyBuild
+    prefix = '/tmp/'
```
Comment (wpoely86) [inline easybuild/easyblocks/generic/pythonpackage.py]: don't hard code this, use the EB provided tmp path.

File: easybuild/easyblocks/generic/pythonpackage.py [hunk 2]
```diff
@@ -53,27 +53,24 @@
...
-        if not txt.startswith(prefix):
-            raise EasyBuildError("Last line of output of %s does not start with specified prefix %s: %s (exit code %s)",
-                                 cmd, prefix, out, ec)
-
-        pylibdir = txt[len(prefix):]
-        log.debug("Determined pylibdir using '%s': %s", cmd, pylibdir)
-        return pylibdir
+
+    # determine Python lib dir via distutils
+    # use run_cmd, we can to talk to the active Python, not the system Python running EasyBuild
```
Comment (wpoely86) [inline easybuild/easyblocks/generic/pythonpackage.py]: shouldn't we at least print out a debugging message about which python is being used?

File: easybuild/easyblocks/generic/pythonpackage.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/generic/pythonpackage.py]: this doesn't create any files, it just queries what the path would look like with this prefix

File: easybuild/easyblocks/generic/pythonpackage.py (hunk 2 above)
Comment (boegel) [inline easybuild/easyblocks/generic/pythonpackage.py]: I guess it doesn't hurt to include it, will do

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**